apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -43,11 +66,24 @@ export function AiGeneratePage({ resLocals }: { resLocals: Record<string, any> }
                   Create question
                 </button>
               </form>
+              Or choose a question from our list of test questions:
+              <select id="user-prompt-prebuilt" onchange="setQuestionToPrebuilt()">
+                <option value=""></option>
+                ${prebuiltQuestions.map((question) => {
+                  return html`<option value="${question.question}">${question.id}</option>`;
+                })}
```

Comment:
Nit, concision:

```suggestion
                ${prebuiltQuestions.map((question) =>
                  html`<option value="${question.question}">${question.id}</option>`
                )}
```

Reply:
Done in 46fe88d71bb20bf3e2763ccd99edba1b5435ea38